packages/hardhat/hardhat.config.ts

```diff
@@ -30,7 +30,7 @@ const config: HardhatUserConfig = {
       },
     },
   },
-  defaultNetwork: "localhost",
```

Comment:
Don't delete localhost as the primary network; just add filecoin as a deployment option & make a comment to replace this with 'calibrationnet' or 'filecoinmainnet'

Reply:
In the verification scripts I added them as `calibration & filecoin` I can also change them to this format !